### ISSUE TITLE
Input is not recognized

### ISSUE THREAD

**Issue by zach-morris** [kodi-game/game.libretro.nestopia#8]
Version tested:  RetroPlayer 18 alpha1 2018-02-11

Input is not recognized.  Below is a log pressing start / A / B.  These inputs on this game did work in the QuickNES core
```
21:57:24.156 T:123145315336192   DEBUG: FEATURE [ start ] on game.controller.default released
21:57:24.156 T:123145315336192   DEBUG: FEATURE [ start ] on game.controller.nes released
21:57:24.156 T:123145315336192   DEBUG: FEATURE [ start ] on game.controller.default released
21:57:24.481 T:123145315336192   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" pressed
21:57:24.482 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default pressed (ignored)
21:57:24.482 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.nes pressed (ignored)
21:57:24.482 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default pressed (handled)
21:57:24.731 T:123145315336192   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" released
21:57:24.731 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default released
21:57:24.732 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.nes released
21:57:24.732 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default released
21:57:25.031 T:140735305183232   DEBUG: ------ Window Deinit (Pointer.xml) ------
21:57:25.172 T:123145315336192   DEBUG: BUTTON [ 13 ] on "PS(R) Gamepad" pressed
21:57:25.172 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.default pressed (ignored)
21:57:25.172 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.nes pressed (ignored)
21:57:25.172 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.default pressed (handled)
21:57:25.332 T:123145315336192   DEBUG: BUTTON [ 13 ] on "PS(R) Gamepad" released
21:57:25.333 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.default released
21:57:25.333 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.nes released
21:57:25.333 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.default released
21:57:25.970 T:123145315336192   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" pressed
21:57:25.971 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default pressed (ignored)
21:57:25.971 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.nes pressed (ignored)
21:57:25.971 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default pressed (handled)
21:57:26.134 T:123145315336192   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" released
21:57:26.135 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default released
21:57:26.135 T:123145315336192   DEBUG: FEATURE [ b ] on game.controller.nes released
21:57:26.135 T:123145315336192   DEBUG: FEATURE [ a ] on game.controller.default released
```

**Comment by garbear** [comment]
Input works for me after https://github.com/kodi-game/game.libretro.nestopia/pull/9. I'll include this in the next build.

**Comment by zach-morris** [comment]
Confirmed fixed in 18.0-ALPHA1 Git:20180215-276e1a4
